test/browser/msw-api/setup-worker/stop/removes-all-listeners.test.ts

```diff
@@ -15,9 +15,12 @@ test('removes all listeners when the worker is stopped', async ({
   fetch,
 }) => {
   const consoleSpy = spyOnConsole()
-  await loadExample(require.resolve('./removes-all-listeners.mocks.ts'), {
-    skipActivation: true,
-  })
+  await loadExample(
+    new URL('./removes-all-listeners.mocks.ts', import.meta.url),
```

Comment:
Not: How stupid it would be to just pass an example name here and resolve it INSIDE loadExample? Quite a lot of repetition here.

Reply:
`import.meta.url` will point to a different path if it's called in `loadExample`. How would you get the path of the module that calls `loadExample()`? Sounds like much more logic than defining a simple URL... But I might be not seeing what you're seeing.